src/interface/src/app/home/plan-table/plan-table.component.ts

```diff
@@ -70,10 +70,12 @@ export class PlanTableComponent implements OnInit {
       return;
     }
     const planIdsToDelete: string[] = [String(this.selectedPlan.id)];
-    const dialogRef: MatDialogRef<DeletePlanDialogComponent> = this.dialog.open(
-      DeletePlanDialogComponent,
+    const dialogRef: MatDialogRef<DeleteDialogComponent> = this.dialog.open(
+      DeleteDialogComponent,
       {
-        data: planIdsToDelete,
+        data: {
+          item: '"' + this.selectedPlan.name + '"',
```

Comment:
just a nit:
 If we're only passing a name here for scenarios and plans, could we just call this attribute "name"? I think it might be clearer, especially if, in the future, we ever wanted to add features to the DeleteDialogComponent to display anything besides just a name.

Reply:
good call!